package.json

```diff
@@ -71,7 +73,13 @@
     "gulp-babel": "6.1.2",
     "gulp-bump": "2.7.0",
     "gulp-eslint": "3.0.1",
+    "jasmine": "2.5.3",
     "karma": "1.6.0",
+    "karma-chrome-launcher": "2.0.0",
+    "karma-jasmine": "1.1.0",
+    "karma-mocha-reporter": "2.2.3",
+    "karma-sourcemap-loader": "0.3.7",
+    "karma-webpack": "2.0.3",
     "lite-server": "2.3.0",
     "mocha": "3.2.0",
```

Comment:
I'm using all of them, e.g. here https://github.com/wholesale-design-system/components/pull/58/files#diff-a2a3b7b0c9c3b4b93b4aebf4e3ec3cfbR33 .

I'm using webpack etc. to build the application and run it in chrome. mocha-reporter because I like the output more then the default jasmine stuff.